src/app/qgsapplayertreeviewmenuprovider.cpp

```diff
@@ -897,7 +896,10 @@ void QgsAppLayerTreeViewMenuProvider::editSymbolLegendNodeSymbol( const QString
 
   const QgsSymbol *originalSymbol = node->symbol();
   if ( !originalSymbol )
+  {
+    QMessageBox::information( nullptr, tr( "No Symbol" ), tr( "There is no symbol associated with the rule." ) );
```

Comment:
Could we use the QgisApp::instance()->messageBar() instead?

Reply:
Done.